docs/resources/boundary_cluster.md

```diff
@@ -0,0 +1,59 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "hcp_boundary_cluster Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  This resource allows you to manage an HCP Boundary cluster
+---
+
+# hcp_boundary_cluster (Resource)
+
+This resource allows you to manage an HCP Boundary cluster
+
+## Example Usage
+
+```terraform
+resource "hcp_boundary_cluster" "example" {
+  cluster_id = "boundary-cluster"
+  username   = "test-user"
+  password   = "Password123!"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `cluster_id` (String) The ID of the Boundary cluster
+- `password` (String, Sensitive) The password of the initial admin user
```

Comment:
👋 Hello,
This is the first time any of the HCP resources are asking the user for a password in terraform. 🤔 
Could you add some type of notice to the user about how the input is stored in the state file or logs?
Similar to: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#password

Reply:
Good catch! I just made some changes to the resource and made note of how the password could show up in logs and is stored in the state file.